src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -747,8 +747,7 @@ OverlayImpl::onManifests (
             {
                 // Historical manifests are sent on initial peer connections.
                 // They do not need to be forwarded to other peers.
-                std::set<Peer::id_t> peers;
-                hashRouter.swapSet (hash, peers, SF_RELAYED);
+                hashRouter.prepRelay (hash);
```

Comment:
Oversight on my part. I was only considering that the local `peers` wasn't used, so switched to the simpler overload.